package.json

```diff
@@ -15,7 +14,9 @@
     "@babel/code-frame": "7.0.0-beta.31",
     "@babel/traverse": "7.0.0-beta.31",
     "@babel/types": "7.0.0-beta.31",
-    "babylon": "7.0.0-beta.31"
+    "babylon": "7.0.0-beta.31",
+    "eslint-scope": "~3.7.1",
+    "eslint-visitor-keys": "^0.1.0"
```

Comment:
@mysticatea Should this be edited to ^1.0.0?

Reply:
@platinumazure I just pushed a commit to update this.